src/proc_image_processing/cpu/filters/filter.h

```diff
@@ -52,6 +63,12 @@ namespace proc_image_processing {
 
         // Useful to identify the filter.
         std::string name_;
+
+    private:
+        virtual void apply(cv::Mat &image) = 0;
+
+        Parameter<bool> enable_{"Enable", false, &parameters_};
+        Parameter<bool> execTime_{"ExecTime", false, &parameters_};
```

Comment:
Use Execution time as the parameter name (this is displayed directly in the telemetry UI)